### ISSUE TITLE
Throwing AdyenException for successful response

### ISSUE THREAD

**Issue by cangelis** [Adyen/adyen-php-api-library#176]
**PHP version**: 7.3
**Library version**: 6.0.1
**Description**

Adyen library is throwing the AdyenException for successful api results in `transferFunds` calls.

See the stacktrace.

```
Adyen\AdyenException: {"pspReference":"4215852201976584","resultCode":"Received","merchantReference":"RefundFee_4515852201309999"} in /tmp/vendor/adyen/php-api-library/src/Adyen/HttpClient/CurlClient.php:289
Stack trace:
#0 /tmp/vendor/adyen/php-api-library/src/Adyen/HttpClient/CurlClient.php(97): Adyen\HttpClient\CurlClient->handleResultError('{"pspReference"...', Object(Monolog\Logger))
#1 /tmp/vendor/adyen/php-api-library/src/Adyen/Service/AbstractResource.php(83): Adyen\HttpClient\CurlClient->requestJson(Object(Adyen\Service\Fund), 'https://cal-liv...', Array, NULL)
#2 /tmp/vendor/adyen/php-api-library/src/Adyen/Service/Fund.php(113): Adyen\Service\AbstractResource->request(Array)
#3 /var/task/app/Services/Adyen.php(161): Adyen\Service\Fund->transferFunds(Array)
```

**Comment by cyattilakiss** [comment]
Hi @cangelis ,

Thank you for raising this issue. I will create the ticket for ourselves and create the fix asap.

Best,
Attila
Adyen

**Comment by cyattilakiss** [comment]
Hi @cangelis ,

We are going to release the fix in version 6.1.0. I will close this issue for now but please feel free to reopen if the issue is still there.

Best,
Attila
Adyen
